package.json

```diff
@@ -14,6 +14,7 @@
   ],
   "scripts": {
     "preinstall": "npx only-allow pnpm",
+    "postinstall": "simple-git-hooks",
```

Comment:
This will make `simple-git-hooks` run every time `pnpm i` is called. But I think it is good to have it for new contributors.
Ideally it is better to turn off [side-effects-cache](https://pnpm.io/npmrc#side-effects-cache) only for `simple-git-hooks`. I'm not sure if we can do this.

Reply:
See https://github.com/toplenboren/simple-git-hooks/issues/34

Reply:
Perhaps we can include it for now until an issue arise, otherwise we can point users to run it manually in the contributing guide.